<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErosionDilateDlg</class>
 <widget class="QDialog" name="ErosionDilateDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxLimit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>406</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>腐蚀膨胀门限</string>
   </property>
   <widget class="QPushButton" name="btnLimitAdd">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>61</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimitSub">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>61</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderLimit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>71</y>
      <width>221</width>
      <height>19</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelLimit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>720</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxedEdgeCalcu">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>边沿提取</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxedSideDelEnable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>130</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>边界删除使能</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxModeType">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>185</y>
     <width>321</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>模板类型</string>
   </property>
   <widget class="QRadioButton" name="radioButton3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3*3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>5*5</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>7*7</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>9*9</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>11*11</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton13">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>13*13</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExpant">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>160</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>高级</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxNum">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>321</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>膨胀/腐蚀次数</string>
   </property>
   <widget class="QPushButton" name="btnNumAdd">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNumSub">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNum">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>35</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxMode">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>腐蚀膨胀模式</string>
   </property>
   <widget class="QRadioButton" name="radioButtonErosion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>腐蚀</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonDilate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>膨胀</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>720</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(108, 108, 108);</string>
   </property>
   <property name="text">
    <string>腐蚀膨胀</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
